app/adapters/alma_adapter.rb

```diff
@@ -13,14 +15,51 @@ def get_bib_record(id)
     get_bib_records([id])&.first
   end
 
+  def build_alma_error_from(json:)
```

Comment:
There are several methods with similar names. Can you add yarddocs to them to make them easier to distinguish and also clarify when each would be used?